eng/synchronize-secrets.ps1

```diff
@@ -0,0 +1,10 @@
+param(
```

Comment:
This entire script is basically just a foreach... is the script really necessary? The call could just be:
`dir .vault-config/*.yaml |% { dotnet run -p }`
The script indirection really isn't adding much.

Reply:
And then you don't hvae to deal with the weird string escaping, or the always scary "Invoke-Expression"

Reply:
yea, thats true. I'll do that.

Reply:
When I first wrote It I thought the script would have more in it.